src/js/media/html5.js

```diff
@@ -26,6 +32,33 @@ vjs.Html5 = vjs.MediaTechController.extend({
       this.setSource(source);
     }
 
+    if (this.el_.hasChildNodes()) {
```

Comment:
Actually, no, we currently can't move this piece down because it depends on the initialization of the 'featuresNativeTexttracks' and emulated text tracks which happen slightly after `createEl` gets called.